src/java.base/share/classes/java/lang/classfile/Annotation.java

```diff
@@ -94,6 +96,8 @@ default ClassDesc classSymbol() {
      */
     static Annotation of(Utf8Entry annotationClass,
                          List<AnnotationElement> elements) {
+        requireNonNull(annotationClass);
+        requireNonNull(elements);
```

Comment:
How did you test these? Shouldn't null elements result in an NPE in `List.copyOf`?